test_submission_schema/v1.json

```diff
@@ -0,0 +1,167 @@
+                },
+                "version": {
+                    "type": "string",
+                    "description": "The version of the application used to run the test plan."
+                },
+                "packaging": {
+                    "type": "object",
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "description": "The type of application. e.g. Snap, Deb."
+                        },
+                        "name": {
+                            "type": "string",
+                            "description": "The name of the application."
+                        },
+                        "version": {
+                            "type": "string",
+                            "description": "The version of the application."
+                        },
+                        "revision": {
+                            "type": "string",
+                            "description": "The revision of the application."
+                        },
+                        "date": {
+                            "type": "string",
+                            "format": "date",
+                            "description": "The package creation date."
+                        }
+                    },
+                    "if": {"properties": {"type": {"const": "snap"}}},
+                    "then": {"required": ["revision"]}
+                }
+            }
+        },
+        "session_data": {
+            "type": "object",
+            "required": [
+                "title",
+                "test_plan_id"
+            ],
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "description": "The title of the test plan."
+                },
+                "test_plan_id": {
+                    "type": "string",
+                    "pattern": "^[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+::[a-zA-Z0-9_]+$",
+                    "description": "The unique identifier of the test plan."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "A brief description of the test plan."
+                },
+                "execution_id": {
+                    "type": "string",
+                    "description": "Test Observer execution ID."
+                }
+            }
+        },
+        "results": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": [
+                    "id",
+                    "test_description",
+                    "certification_status",
+                    "category_id",
+                    "outcome",
+                    "comments",
+                    "io_log",
+                    "duration",
+                    "type",
+                    "template_id"
```

Comment:
This is probably only required when running an instance of a template? Thus, optional?

Reply:
Changed to `test_group_id`